api/controllers/AttendanceController.ts

```diff
@@ -25,16 +29,26 @@ export class AttendanceController {
     return { error: null, attendances };
   }
 
+  @UseBefore(UserAuthentication)
   @Get()
   async getAttendancesForCurrentUser(@AuthenticatedUser() user: UserModel): Promise<GetAttendancesForUserResponse> {
     const attendances = await this.attendanceService.getAttendancesForUser(user);
     return { error: null, attendances };
   }
 
+  @UseBefore(UserAuthentication)
```

Comment:
Is this decorator related to the current PR or a bug fix?

Reply:
Current PR - previously we've had this decorator over the entire AttendanceController class, meaning that every route is authenticated. This changes in this PR because this new route isn't authenticated, so we need to remove the decorator over the whole class and decorate only the routes that are authenticated.